packages/@uppy/locales/src/en_US.js

```diff
@@ -30,6 +30,8 @@ en_US.strings = {
   companionError: 'Connection with Companion failed',
   companionUnauthorizeHint: 'To unauthorize to your %{provider} account, please go to %{url}',
   complete: 'Complete',
+  compressedX: 'Saved %{size} by compressing images',
```

Comment:
I wonder if the string should actually refer to images, while the key could imply it can be used to report a compression of any kind.

Reply:
Good point, although already a breaking change I suppose if it's released.